src/main/java/io/github/terra121/projection/GeographicProjection.java

```diff
@@ -277,4 +277,42 @@ public float azimuth(double x, double y, float angle) throws OutOfProjectionBoun
 	public enum Orientation {
 		none, upright, swapped
 	}
+	public enum Facing {
+		N (337.5, 360, "North"),
+		N2 (0, 22.5, "North"),
+		S (157.5, 202.5, "South"),
+		E (67.5, 112.5, "East"),
+		W (247.5, 292.5, "West"),
+		NW (292.5, 337.5, "Northwest"),
+		SW (202.5, 247.5, "Southwest"),
+		SE (112.5, 157.5, "Southeast"),
+		NE (22.5, 67.5, "Northeast"),
+		Unknown (360.5, 10000000.0, "Unknown");
+	    private final double min;
+	    private final double max;
+	    private final String realName;
+	    private Facing(double min, double max, String realName) {
+	        this.min = min;
+	        this.max = max;
+	        this.realName = realName;
+	    }
+	    public double getMin() {
+	    	return min;
+	    }
+	    public double getMax() {
+	    	return max;
+	    }
+	    public String getRealName() {
+	    	return realName;
+	    }
+    }
+	public static String azimuthToFacing(float azimuth) {
+		for (Facing facingToBeTestedFor : Facing.values()) {
+			if (azimuth >= facingToBeTestedFor.getMin() && azimuth <= facingToBeTestedFor.getMax()) {
```

Comment:
Testing if  an azimuth matches a specific cardinal direction is something that could be useful outside of this loop, it would be nice to have implemented as a method.

Reply:
Sure